specification/applicationinsights/ApplicationInsights.LiveMetrics/routes.tsp

```diff
@@ -0,0 +1,41 @@
+import "@azure-tools/typespec-azure-core";
+import "@typespec/rest";
+import "@typespec/http";
+import "./models.tsp";
+
+using Azure.Core;
+using Azure.Core.Traits;
+using TypeSpec.Rest;
+using TypeSpec.Http;
+using TypeSpec.Versioning;
+
+namespace LiveMetrics;
+
+op QuickPulseOperation<
+  TParams extends TypeSpec.Reflection.Model,
+  TResponse extends TypeSpec.Reflection.Model
+> is Foundations.Operation<TParams, TResponse, {}, ServiceError>;
+
+#suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Doesn't fit standard ops"
+@doc("Client SDK calls QuickPulse /ping API.")
+@route("/QuickPulseService.svc/ping")
+@post
+op ping is QuickPulseOperation<
```

Comment:
Need better method names and documentation. These become the client SDK method names, so something like `ping()` or `post()` aren't very helpful. Also, you don't want the method docs mentioning "Client SDK calls QuickPulse /ping API.". Think about other SDK method names and docs.